app/src/sharedTest/java/org/oppia/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -160,38 +163,64 @@ class TopicPracticeFragmentTest {
+      ).perform(click())
+      onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).check(matches(isChecked()))
     }
-    activityScenario.recreate()
-    onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).check(matches(isChecked()))
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicPracticeFragment_loadFragment_configurationChange_startButtonRemainsInactive() {
-    onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
-    activityScenario.onActivity { activity ->
-      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
-    }
-    activityScenario.recreate()
-    onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
-    onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))    }
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicPracticeFragment_loadFragment_selectSkills_configurationChange_startButtonRemainsActive() {
-    onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).perform(click())
-    activityScenario.onActivity { activity ->
-      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(isClickable()))    }
+  }
+
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeorientation_titleiscorrect() {
```

Comment:
Name is incorrect. We are following camelCase convention separated by `_`. That means that `changeorientation` should be `changeOrientation` and `titleiscorrect` should be `titleIsCorrect`.
`testTopicPracticeFragment_loadFragment_changeOrientation_titleIsCorrect`

Reply:
Done:)